src/main/resources/application.properties

```diff
@@ -1,9 +1,13 @@
 # H2
 spring.h2.console.enabled=true
 spring.h2.console.path=/h2
+
 # Datasource
 spring.datasource.url=jdbc:h2:file:~/website-be-h2-db
 spring.datasource.username=h2dev
 spring.datasource.password=
 spring.datasource.driver-class-name=org.h2.Driver
 spring.jpa.hibernate.ddl-auto=update
+
+# API Settings
+server.error.whitelabel.enabled=false
```

Comment:
What is this for? I couldn't understand the difference when removing this

Reply:
This makes the server not present the default HTML error page for undefined endpoints, instead falling back to the Apache Tomcat 404 page. The latter behavior is disabled, though, by the `ErrorController`, which sends a json containing "Invalid endpoint". (so, in practice, it has no effect for us, but makes sense to me since we're building an API)